api-interop-layer/data/alerts/index.test.js

```diff
@@ -1,6 +1,25 @@
 import sinon from "sinon";
 import { expect } from "chai";
 import * as mariadb from "mariadb";
+import dayjs from "../../util/day.js";
+import { alignAlertsToDaily } from "./utils.js";
+
+/**
+ * Test helper for creating arrays of mock
+ * day / hours objects
+ */
+const makeDayWithHours = (startTimestamp, numHours, timezone="America/New_York") => {
+  const hours = [];
+  const start = dayjs.utc(startTimestamp).tz(timezone);
+  // eslint-disable-next-line no-plusplus
+  for(let i = 0; i < numHours; i++){
```

Comment:
```suggestion
  for(let i = 0; i < numHours; i += 1){
```
I don't know why eslint prefers `+= 1` over `++`, but that's what it wants. 🤷